### PR TITLE
Remove the requirement for a rubocop submodule

### PR DESCRIPTION
**Pull request by tjwp**
The latest RuboCop version (v0.41.1) exposes files to support testing with RSpec.

### CODE REVIEW

**Comment by tjwp** [comment]
Updated now that 0.41.1 is out.

**Comment by tjwp** [comment]
The problem with `rubocop:disable` in 0.41.1 worked around here should be fixed by https://github.com/bbatsov/rubocop/pull/3250.

**Comment by tjwp** [comment]
Updated again now that 0.41.2 has been released which allows RSpec cops to be enable/disabled by comments. 

Also, fixed a test issue with ruby 1.9 now that json 2.0 depends on ruby 2.0 or later.

**Comment by backus** [comment]
Thanks @tjwp. Could you rebase onto master? I'd like to merge this

**Comment by tjwp** [comment]
@backus rebased

**Comment by backus** [comment]
Thanks @tjwp!

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,8 @@ gemspec
 
 group :test do
   gem 'codeclimate-test-reporter', require: false
+  # json 2.0 requires Ruby >= 2.0
+  gem 'json', '~> 1.8', require: false, platforms: :ruby_19
```
Comment (backus) [inline Gemfile]: You should be able to remove this. I dropped 1.9.3 support.

**Comment by backus** [comment]
@tjwp I left one comment. I'll merge once that is fixed

**Comment by tjwp** [comment]
@backus done

**Comment by backus** [comment]
Thanks again @tjwp!

**Pull request merged**